docs/articles/index.html

```diff
@@ -0,0 +1,140 @@
+    </div>
+    <div class="main-content">
+        <div class="feature">
+            <span class="name">
+                Easy Install
+            </span>
+            <div class="description">
+                <p>All you need is to have <a href="https://nodejs.org">Node.js</a> with <a href="https://www.npmjs.com">npm</a> installed and call a single command.</p>
+                <div class="command">npm install --save-dev testcafe</div>
+                <p>No browsers plugins to install, no binary compilation post-install steps.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Complete Test Harness
+            </span>
+            <div class="description">
+                <p>TestCafe covers all testing phases: starting browsers, running tests and gathering results. You configure test run and start execution either via the API or from a command shell simply by running a single command</p>
+                <div class="command">testcafe safari,ie tests/</div>
+                <p>TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Test Code Using ES2016
+            </span>
+            <div class="description">
+                <p>You can write TestCafe tests in ES2016 using all latest JavaScript features like 'async/await'.</p>
+                <p>TestCafe introduces simple, but powerful test API. It offers a couple of dozen methods covering all user actions. Chained syntax produces code that is easy to write and read. Furthermore, you are free to use any assertion library to perform verifications of different kinds.</p>
+                <pre><code>import { expect } from 'chai';
+
+fixture `My Fixture`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My Test', async t => {
+    await t
+        .click('#send-button')
+        .handleAlert()
+        .typeText('#input', 'Peter Parker')
+        .wait(1000);
+
+    expect(await t.eval(() => getSomethingOnTheClient())).to.be.true;
+});</code></pre>
+                <p>TestCafe will compile your test code on-flight and run it immediately. It also ships with built-in support for source maps to leverage debugging experience. Source maps are automatically enabled, so all you need to do is start a debugging session in an IDE that supports them.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Regular Client JavaScript to Observe Page State
+            </span>
+            <div class="description">
+                <p>TestCafe is capable of executing code on the client side thus giving you direct access to DOM elements on the page and powering you to obtain some data from the client.
```

Comment:
> powering you to obtain some data from the client.

powering you to obtain **specific** [or probably better no adjective at all] data from the client.

'some' is no good